eos/utils/integrate.cc

```diff
@@ -281,6 +282,48 @@ namespace eos
         return result;
     }
 
+    namespace cubature
+    {
+        Config::Config() :
+            _qng(),
```

Comment:
Hrm. I know why you're doing this, but it still rubs me in a wrong way. Can we rename ```QNG::Config``` to something like ```PrecisionGoal```, and use a composite for a new ```QNG::Config```? I think that makes more sense.

Reply:
Sure, and it feels slightly odd. I'll refactor